src/pages/publication/ContributorsPanel.tsx

```diff
@@ -1,20 +1,67 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback, useEffect } from 'react';
```

Comment:
får en warning i nettleseren fra addcontributormodalcontent.tsx : 
 Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in AuthorityCard (at AddContributorModalContent.tsx:103)
    in div (created by styled.div)
    in styled.div (at AddContributorModalContent.tsx:99)

Reply:
ja, det er pga det fortsatt hentes data fra ALMA etter man har lukket modalen. jeg ser for meg at dette fikses når vi får på plass ALMA-proxy (for da er det lettere å kansellere requesten hvis modalen lukkes)